src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDriver.java

```diff
@@ -644,7 +644,9 @@ enum SQLServerDriverIntProperty {
     STATEMENT_POOLING_CACHE_SIZE("statementPoolingCacheSize", SQLServerConnection.DEFAULT_STATEMENT_POOLING_CACHE_SIZE),
     CANCEL_QUERY_TIMEOUT("cancelQueryTimeout", -1),
     CONNECT_RETRY_COUNT("connectRetryCount", 1, 0, 255),
-    CONNECT_RETRY_INTERVAL("connectRetryInterval", 10, 1, 60);
+    CONNECT_RETRY_INTERVAL("connectRetryInterval", 10, 1, 60),
+    BULK_COPY_OPTION_DEFAULTS_BATCH_SIZE("bulkCopyOptionDefaultsBatchSize", 0),
```

Comment:
Could the method name be BULK_COPY_OPTION_BATCH_SIZE/BULK_COPY_BATCH_SIZE instead of BULK_COPY_OPTION_DEFAULTS_BATCH_SIZE? (If it is fine then we need to use same pattern in all methods and constants)